frontend/containers/Admin/UsersDetail.js

```diff
@@ -36,10 +36,12 @@ class AdminUsersDetail extends Component {
 
   loadUsers () {
```

Comment:
Can you move load users from the constructor to ComponentDidMount? This should fix the set state issue